src/main/kotlin/no/nav/syfo/sykmeldingstatus/kafka/producer/BekreftSykmeldingKafkaProducer.kt

```diff
@@ -0,0 +1,15 @@
+package no.nav.syfo.sykmeldingstatus.kafka.producer
+
+import no.nav.syfo.sykmeldingstatus.kafka.model.SykmeldingKafkaMessage
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerRecord
+
+class BekreftSykmeldingKafkaProducer(private val kafkaProducer: KafkaProducer<String, SykmeldingKafkaMessage?>, private val topic: String) {
+    fun sendSykmelding(sykmeldingKafkaMessage: SykmeldingKafkaMessage) {
+        kafkaProducer.send(ProducerRecord(topic, sykmeldingKafkaMessage.sykmelding.id, sykmeldingKafkaMessage))
+    }
+
+    fun tombstoneSykmelding(sykmeldingId: String) {
```

Comment:
👏 

Legger du til dette for sendt-topicen også?

Reply:
Eller, ser nå at dette er for å nullstille tidligere bekreftinger, det er jo kanskje ikke så aktuelt for sendte sykmeldinger 😅 Men det kan hende vi trenger å kunne tombstone for å kunne slette ting uansett da, men det kan vi ta når vi begynner å se på sletting evt